timesketch/frontend-ng/src/components/Graph/Cytoscape.vue

```diff
@@ -0,0 +1,717 @@
+
+    <!-- Event list for selected edges -->
+    <v-bottom-sheet hide-overlay persistent no-click-animation v-model="showTimelineView" scrollable>
+      <v-card>
+        <v-toolbar dense flat>
+          <strong>Timeline for {{ selectedEdgesCount }} selected edge(s)</strong>
+          <v-spacer></v-spacer>
+          <v-btn icon :disabled="timelineViewHeight > 40" @click="increaseTimelineViewHeight()">
+            <v-icon>mdi-chevron-up</v-icon>
+          </v-btn>
+          <v-btn icon :disabled="timelineViewHeight === 0" @click="decreaseTimelineViewHeight()">
+            <v-icon>mdi-chevron-down</v-icon>
+          </v-btn>
+          <v-btn icon @click="showTimelineView = false">
+            <v-icon>mdi-close</v-icon>
+          </v-btn>
+        </v-toolbar>
+        <v-divider></v-divider>
+        <v-expand-transition>
+          <v-card-text :style="{ height: timelineViewHeight + 'vh' }" v-show="!minimizeTimelineView">
+            <ts-event-list
+              :query-request="queryRequest"
+              :items-per-page="maxEvents"
+              disable-save-search
+            ></ts-event-list>
+          </v-card-text>
+        </v-expand-transition>
+      </v-card>
+    </v-bottom-sheet>
+  </v-container>
+</template>
+
+<script>
+import ApiClient from '../../utils/RestApiClient'
+import EventBus from '../../main'
+import cytoscape from 'cytoscape'
+import spread from 'cytoscape-spread'
+import dagre from 'cytoscape-dagre'
+
+import TsEventList from '../Explore/EventList'
+
+cytoscape.use(spread)
+cytoscape.use(dagre)
+
+export default {
+  props: {
+    graphPluginName: {
+      type: String,
+      default: null,
+    },
+    savedGraphId: {
+      type: Number,
+      default: null,
+    },
+    canvasHeight: {
+      type: String,
+      default: '85vh',
+    },
+    disableZoom: {
+      type: Boolean,
+      default: false,
+    },
+  },
+  components: {
+    TsEventList,
+  },
+  data: function () {
+    return {
+      saveGraphDialog: false,
+      graphSettingsMenu: false,
+      showTimelineView: false,
+      timelineViewHeight: 40,
+      minimizeTimelineView: false,
+      isLoading: false,
+      filterString: '',
+      graphs: {},
+      currentGraph: '',
+      currentGraphCache: {},
+      currentGraphCacheConfig: {},
+      fadeOpacity: 7,
+      elements: [],
+      selectedEdgesCount: 0,
+      edgeQuery: '',
+      maxEvents: 40,
+      queryFilter: {
+        from: 0,
+        terminate_after: 40,
+        size: this.maxEvents,
+        indices: ['_all'],
+        order: 'asc',
+        chips: [],
+      },
+      saveAsName: '',
+      layouts: ['spread', 'dagre', 'circle', 'concentric', 'breadthfirst'],
+      layoutName: 'spread',
+      edgeStyles: ['bezier', 'taxi'],
+      edgeStyle: 'bezier',
+      config: {
+        style: [
+          {
+            selector: 'node',
+            style: {
+              shape: 'roundrectangle',
+              width: 'label',
+              height: 'label',
```

Comment:
According to the warning on my console the style value of `label` is deprecated for width and height in cytoscape.
The recommended alternative I found is to measure the length of your test and then use this to determine the size.

Reply:
Yeah, I opted to followup in another PRs for that. Tracking in: https://github.com/google/timesketch/issues/2743